### PR TITLE
Update create site:

### PR DESCRIPTION
**Pull request by jdrossl**
- remote name parameter is now optional, defaults to `origin`

https://github.com/craftercms/craftercms/issues/3479

### CODE REVIEW

File: src/main/java/org/craftercms/studio/api/v2/utils/StudioConfiguration.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
     String SITES_REPOS_PATH = "studio.repo.sitesRepoBasePath";
     String SANDBOX_PATH = "studio.repo.siteSandboxPath";
     String REPO_SANDBOX_BRANCH = "studio.repo.siteSandboxBranch";
+    String REPO_REMOTE_NAME = "studio.repo.siteRemoteName";
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/api/v2/utils/StudioConfiguration.java]: Add the word DEFAULT.

Maybe: REPO_DEFAULT_REMOTE_NAME

File: src/main/resources/crafter/studio/studio-config.yaml [hunk 2]
```diff
@@ -25,6 +25,8 @@ studio.repo.sitesRepoBasePath: sites
 studio.repo.siteSandboxPath: sandbox
 # Sandbox git repository branch for every site
 studio.repo.siteSandboxBranch: master
+# Name used for the git remote for remote sites
+studio.repo.siteRemoteName: origin
```
Comment (sumerjabri) [inline src/main/resources/crafter/studio/studio-config.yaml]: studio.repo.defaultRemoteName

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Should we also update the docs to indicate that this param is now optional?

**Comment by jdrossl** [comment]
PR for the doc https://github.com/craftercms/docs/pull/849

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**